website/tables/gcp/gcp_network_connectivity_internal_ranges.md

```diff
@@ -0,0 +1,28 @@
+# Table: gcp_network_connectivity_internal_ranges
+
+This table shows data for GCP Network Connectivity Internal Ranges.
+
+https://cloud.google.com/network-connectivity/docs/reference/networkconnectivity/rest/v1/projects.locations.internalRanges/list
+
+The primary key for this table is **name**.
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_id|`uuid`|
+|_cq_parent_id|`uuid`|
+|project_id|`utf8`|
```

Comment:
I think the PK should also include proj ID & location

Reply:
I don't think `project_id` would be unique, as the single project could have multiple internal ranges. and location is not part of the API response. The `name` has the combination of both `project_id` and `location` .

Ex: projects/infoblox-saas/locations/global/internalRanges/ir-2

Reply:
I only suggested adding `project_id` & `location_id` to the PK, so that PK would be `project_id, location_id, name`, but `name` also works here (per reading `name` format).
However, `project_id` & `location_id` can be viewed here as `request project_id` & `request location_id` respectively (if these ranges will ever be shared/viewable from different points).

BTW, did you test that `global` is returned as a location ID in list locations, too?

Reply:
yes I have verified that APIs returns `location_id`:`global` and corresponding name as `projects/<project-ID>/locations/global`